config/locales/en/pages.yml

```diff
@@ -1,5 +1,31 @@
 en:
   pages:
+    "404":
+      title: This page cannot be found
+      body_html: |
+        <p>You can <a href="/">start a new prison visit request</a> or book
+        your visit by phone instead. See the
+        <a href="http://www.justice.gov.uk/contacts/prison-finder">prison
+        finder</a> for contact details.</p>
+    "500":
+      title: We are experiencing technical difficulties
+      body_html: |
+        <p>The prison visits booking system is not available right now, but
+        we're working hard to get things up and running again.</p>
+
+        <p>You can try again later or book your visit by phone instead. See the
+        <a href="http://www.justice.gov.uk/contacts/prison-finder">prison
+        finder</a> for contact details.</p>
+    "503":
+      title: We are experiencing technical difficulties
+      body_html: |
+        <p>The prison visits booking system is not available as we're carrying
+        out some planned maintenance. You can try again later or book your
+        visit by phone instead.</p>
+
+        <p>You can try again later or book your visit by phone instead. See the
+        <a href="http://www.justice.gov.uk/contacts/prison-finder">prison
+        finder</a> for contact details.</p>
```

Comment:
@threedaymonk I'm not convinced that separating the paragraphs is useful thought – surely having one or two paragraphs is a copy decision and it's best not to couple to the template. Personally I'd rather content editors were able to write markdown rather than HTML in this file.